requirements_test.txt

```diff
@@ -17,3 +17,4 @@ torch>=1.10.1
 torchvision>=0.11.2
 zennit>=0.4.5 ; python_version > '3.6'
 tqdm>=4.62.3
+pytest-mock==3.8.2
```

Comment:
What benefits do pytest-mock have?

Reply:
`test_tf_model.test_predict` asserts against concrete values.
The test checks if `quantus.TensorFlowModel` wrapper correctly applies softmax or linear activations. However, the actual output values depend on the output of the underlying model, which is not part of Quantus codebase. So, I just mock the `tf.keras.predict`.
Afaik, it is common practice to use mocks, when doing asserts on component, which depend on 3-rd party components.

Reply:
Nice. Thanks.